tests/apollo/util/bft.py

```diff
@@ -714,19 +734,24 @@ def start_replica_cmd(self, replica_id):
             keys_path = os.path.join(self.txn_signing_keys_base_path, "transaction_signing_keys")
             cmd.append("-t")
             cmd.append(keys_path)
-        return cmd
+        return cmd, cmd[replica_binary_path_index]
```

Comment:
This function is implemented by multiple test suites and returns a single object, which makes this default one obscure.
Why do you need two return values?
In case this change is kept, the function's documentation should be updated with the reason.